lib/validation/validators/decimal.ts

```diff
@@ -0,0 +1,5 @@
+const regExp = /^[-]?\d*(\.\d+)?$/
+
+export function decimal(value: string): boolean {
+  return regExp.test(value)
+}
```

Comment:
The code of this validator is copied from Vuelidate:

- https://github.com/vuelidate/vuelidate/blob/08aa51495a4590c17d407a56cbbada9cf370c5d7/packages/validators/src/raw/decimal.js
- https://github.com/vuelidate/vuelidate/blob/08aa51495a4590c17d407a56cbbada9cf370c5d7/packages/validators/src/raw/core.js#L48-L56